app/services/epay_service.rb

```diff
@@ -0,0 +1,59 @@
+module Spree
+    class EpayService
+      def initialize(order, payment_method)
+        @order = order
+        @payment_method = payment_method
+      end
+
+      def payment
+        response = connection.post(api_url, request_body)
+        response.body
+      end
+
+      private
+
+      # Setting to true will log requests and response to $stdout,
+      # should always be set to false when commiting code
+      DEBUG_MODE = true
```

Comment:
@shradhaN lets utilise debug level set on development.rb, production.rb..
is debug_level is warning, we can enable DEBUG_MODE..